app.js

```diff
@@ -36,6 +36,8 @@ options:
   --version                    Print the current version of Scandium, then exit.
   --output=<path>              Output built zip. Example: "--output scandium.zip"
   --bucket=<name>              Use S3 bucket for deployment, needed for deployments over 50MB.
+  --handler=<name>             Use custom handler instead of scandium http handler
+  --worker                     Deploy lambda function as worker (no http)
```

Comment:
Could we combine these options somehow? 🤔 

I cannot think of a use case for using only one of them without the other, or am I missing something?

I'm also not sure about the name "worker", maybe just call it `--no-api-gateway`.

Hmmmmm, or we could have just `--worker=<handler>`, which would skip API Gateway, not add the `scandium-entrypoint.js` file, and set the handler to a custom value.

🤔